modules/apps/frontend-js/frontend-js-svg4everybody-web/src/main/java/com/liferay/frontend/js/svg4everybody/web/internal/servlet/taglib/SVG4EverybodyTopHeadDynamicInclude.java

```diff
@@ -83,9 +104,12 @@ protected void activate(BundleContext bundleContext) {
 	}
 
 	private static final String[] _JS_FILE_NAMES = {
-		"/svg4everybody/svg4everybody.js"
+		"/index.js"
```

Comment:
@jbalsas: Is this simplification I did here kosher? Just to make the URL and the filesystem cleaner... browser still requests `/o/frontend-js-svg4everybody-web/index.js` so there's no ambiguity what this is when you look at it in the network pane.

Reply:
Looks good enough to me 👍